### PR TITLE
Support for non-transactional write.

### PR DESCRIPTION
**Pull request by Bhupender-Y**
Use non-transactional write processing raw files. In the current scenario for raw files PSC is using transactional write. But in transactional write, there is a limit of event size i.e 8mb. To resolve it out, for raw files processing instead of using transactional writer, it will use non-transactional writer.

### CODE REVIEW

**Review by dada-dell-emc — APPROVED** [review]

**Comment by ShwethaSNayak** [comment]
Changes looks good to me. Elaborated description might help to understand the context.

I assume post changes below scenarios are validated.
1. Raw file with size more than 8mb
2. Raw file with size less than 8mb

**Review by ShwethaSNayak — COMMENTED** [review]
Changes looks good to me. Elaborated description might help to understand the context.

I assume post changes below scenarios are validated.

1. Raw file with size more than 8mb
2. Raw file with size less than 8mb

File: pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileProcessor.java [hunk 1]
```diff
@@ -176,9 +177,14 @@ void processFile(FileNameWithOffset fileNameWithBeginOffset, long firstSequenceN
         /* Check if transactions can be aborted.
          * Will fail with {@link TxnFailedException} if the transaction has already been committed or aborted.
          */
-        log.debug("processFile: Transaction status {} ", writer.getTransactionStatus());
-        if(writer.getTransactionStatus() == Transaction.Status.OPEN){
-            writer.abort();
+        if (config.exactlyOnce)
+        {
```
Comment (sachin-j-joshi) [inline pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileProcessor.java:181]: wrong indentation

**Review by sachin-j-joshi — COMMENTED** [review]
Looks good to me.
Let's test it thoroughly before merging

**Review by sachin-j-joshi — APPROVED** [review]
LGTM

**Pull request merged**